client/my-sites/stats/activity-log-example/style.scss

```diff
@@ -0,0 +1,10 @@
+
+.activity-log-example {
+	margin-bottom: 1px;
+
+	.feature-example {
+		// Make overflowing border visible
+		margin-right: -1px;
+		padding-right: 1px;
```

Comment:
Not a huge fan of this but didn't see any obvious way to fix it directly at `<FeatureExample>`